common/rpc_service.go

```diff
@@ -42,6 +42,32 @@ type RPCService interface {
 	Reference() string
 }
 
+// GetReference return the reference id of the service.
+// If the service implemented the RPCService interface,
+// it will call the Reference method; if not, it will
+// return the struct name as the reference id.
+func GetReference(service interface{}) string {
```

Comment:
// GetReference returns the reference id of the service.
// If the service implemented the RPCService interface,
// it will call the Reference method. If not, it will
// return the struct name as the reference id.

Reply:
I have modified it. 😊